svc.go

```diff
@@ -46,7 +46,7 @@ func newAutoNATService(ctx context.Context, c *config) (*autoNATService, error)
 		reqs:   make(map[peer.ID]int),
 	}
 
-	if c.forceServer {
+	if c.forceServer || c.forceReachability {
```

Comment:
We should probably get rid of the `forceServer` option, right?

Reply:
Also, what if we're forcing "unreachable"? Will we not start in that case?

Reply:
removed. When unreachable, the service object won't be created in `autonat.New`